optking/optimize.py

```diff
@@ -15,8 +29,8 @@
 
 
 def optimize(o_molsys, computer):
-    """Driver for OptKing's optimization procedure. Suggested that users use optimize_psi4 or
-     optimize_qcengine to perform a normal (full) optimization
+    """Driver for OptKing's optimization procedure. Suggested that users use EngineHelper CustomHelper or
```

Comment:
Is black or something removing all the commas?  Sometimes text difficult to read due to this.

Reply:
No it isn't black it's me. I think I've been just been writing quickly and not proofreading carefully enough.